cpp/include/cudf_test/iterator_utilities.hpp

```diff
@@ -38,15 +41,43 @@ namespace test {
  * iter[9] == false; // i.e. Invalid row at index 9.
  * @endcode
  *
- * @param indices The collection of indices for which the validity iterator
- *                must return `false` (i.e. null)
+ * @tparam Iter Iterator type
+ * @param index_start Iterator to start of indices for which the validity iterator
+ *                    must return `false` (i.e. null)
+ * @param index_end   Iterator to end of indices for the validity iterator
+ * @return auto Validity iterator
+ */
+template <typename Iter>
+static auto iterator_with_null_at(Iter index_start, Iter index_end)
+{
+  using index_type = typename std::iterator_traits<Iter>::value_type;
+
+  return cudf::detail::make_counting_transform_iterator(
+    0, [indices = std::vector<index_type>{index_start, index_end}](auto i) {
+      return std::find(indices.cbegin(), indices.cend(), i) == indices.cend();
+    });
+}
+
+/**
+ * @brief Bool iterator for marking (possibly multiple) null elements in a column_wrapper.
+ *
+ * The returned iterator yields `false` (to mark `null`) at all the specified indices,
+ * and yields `true` (to mark valid rows) for all other indices. E.g.
+ *
+ * @code
+ * auto iter = iterator_with_null_at(std::vector<size_type>{8,9});
```

Comment:
docs need to be updated here and above.

Reply:
I'd left the first unchanged since `host_scan`'s constructor is non-explicit, and can accept `std::vector`. But the function template's docs should've been changed.

I'll change both, for clarity.